universal-login-sdk/lib/core/services/WalletService.ts

```diff
@@ -163,29 +179,53 @@ export class WalletService {
 
   disconnect(): void {
     this.stateProperty.set({kind: 'None'});
-    this.removeFromStorage();
+    this.saveToStorage();
   }
 
-  saveToStorage(applicationWallet: ApplicationWallet) {
-    this.storage && this.storage.save(applicationWallet);
+  saveToStorage() {
+    if (!this.storage) {
```

Comment:
Why not throw an exception? What would be the use-case for WalletService without storage?

Reply:
👍

Reply:
In-memory wallet only. We do this in playground for example

Reply:
Shall we then implement an in-memory store as a separate class (for test and playground)?

Reply:
`WalletService` already has it's own internal in-memory state storage. We cannot fully offload the state to injectable storage since it has some unserializable parts. So creating a separate `MemoryWalletStorageService` right now isn't needed since it would have the same data lifetime as `WalletService`'s internal state store.